packages/config/config/devices/0x031e/vzw31-sn.json

```
@@ -0,0 +1,699 @@
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Immediate",
+					"value": 0
+				},
+				{
+					"label": "500 ms",
+					"value": 5
+				},
+				{
+					"label": "1 second",
+					"value": 10
+				},
+				{
+					"label": "2 seconds",
+					"value": 20
+				},
+				{
+					"label": "2.5 seconds",
+					"value": 25
+				},
+				{
+					"label": "3 seconds",
+					"value": 30
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Dimming Speed - Up (Local)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_1"
+		},
+		{
+			"#": "3",
+			"label": "Ramp Rate - Off to On (Remote)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_1"
+		},
+		{
+			"#": "4",
+			"label": "Ramp Rate - Off to On (Local)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_3"
+		},
+		{
+			"#": "5",
+			"label": "Dimming Speed - Down (Remote)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_1"
+		},
+		{
+			"#": "6",
+			"label": "Dimming Speed - Down (Local)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_2"
+		},
+		{
+			"#": "7",
+			"label": "Ramp Rate - On to Off (Remote)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_3"
+		},
+		{
+			"#": "8",
+			"label": "Ramp Rate - On to Off (Local)",
+			"$import": "templates/inovelli_templates.json#dimming_speed_0_255_sync_4"
+		},
+		{
+			"#": "9",
+			"label": "Minimum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 54,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		{
+			"#": "10",
+			"label": "Maximum Level",
+			"valueSize": 1,
+			"minValue": 55,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"unsigned": true
+		},
+		{
+			"#": "11",
+			"label": "Invert Switch",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "12",
+			"label": "Auto Off Timer",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "13",
+			"label": "Default Level (Local)",
+			"$import": "templates/inovelli_templates.json#level_1-99_0_previous"
+		},
+		{
+			"#": "14",
+			"label": "Default Level (Remote)",
+			"$import": "templates/inovelli_templates.json#level_1-99_0_previous"
+		},
+		{
+			"#": "15",
+			"label": "State After Power Restored",
+			"$import": "templates/inovelli_templates.json#level_0-99_100_previous",
+			"defaultValue": 100
+		},
+		{
+			"#": "17",
+			"label": "Load Level Indicator Timeout",
+			"valueSize": 1,
+			"minValue": 0,
+			"unit": "seconds",
+			"maxValue": 11,
+			"defaultValue": 11,
+			"options": [
+				{
+					"label": "Always off",
+					"value": 0
+				},
+				{
+					"label": "Always on",
+					"value": 11
+				}
+			]
+		},
+		{
+			"#": "18",
+			"label": "Active Power Reports",
```

If this is for reports on changes, we typically call this
```suggestion
			"label": "Power Change Report Threshold",
```